modules/ddl-and-loading/pages/running-a-loading-job.adoc

```diff
@@ -85,6 +85,32 @@ For example, `-n 50` means read lines 1 to 50.
 `-n 10, 50` means read lines 10 to 50.
 The special symbol `$` is interpreted as "last line", so `-n 10,$` means reads from line 10 to the end.
 
+==== `-max_percent_error`
+
+The `-max_percent_error` option is used by the loader process to decide when to automatically abort a running loading job.
+This depends on a user provided percentage of the data input lines that are rejected.
+
+There are two possible cases where an input line may be rejected:
+
+* Input format error: This is the case where the input line cannot be parsed properly.
+* Filtering error: This is the case where the input line can be parsed properly, but it does not pass certain filtering rules defined in the loading job itself.
+
+For example, if `-max_percent_error` is set to 50%, and over 50% of data input lines in a given batch are rejected by the loader either due to parsing error or filtering error, the loader process will abort itself.
+
+NOTE: The default percentage is set to 0.
```

Comment:
I recommend removing this line and instead mention the default behavior in the table

Reply:
removed with latest commit.